Documentation/ceph-configuration.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configuration
+weight: 3700
+indent: true
+---
+
+# Configuration
+
+For most any Ceph cluster, the user will want to -- and may need to -- change some Ceph
+configurations. These changes often may be warranted in order to alter performance to meet SLAs or
+to update default data resiliency settings.
+
+**WARNING:** Modify Ceph settings carefully, and review the
+[Ceph configuration documentation](https://docs.ceph.com/docs/master/rados/configuration/) before
+making any changes. Changing the settings could result in unhealthy daemons or even data loss if
+used incorrectly.
+
+## Required configurations
+Rook and Ceph both strive to make configuration as easy as possible, but there are some
+configuration options which users are well advised to consider for any production cluster.
+
+### Default PG and PGP counts
+`osd_pool_default_pg_num` and `osd_pool_default_pgp_num`. The number of PGs and PGPs can be
```

Comment:
Users are going to see issues with PGs surfaced as warnings so I wonder if they will look to the troubleshooting guide first in order to resolve the warnings. What if we put PG configuration in ceph-common-issues.md, and reserve this for the more general topic of how to apply other ceph config options? Or else we keep this topic here and promote it to a larger title so they see that they need to learn how to configure PGs.

Reply:
Waiting until the users might notice an error doesn't seem like the right time to confront this topic. Configuration is a necessary part of any software, and it should be a prominent topic in Rook's docs. I'm not entirely sure I understand what you are suggesting by promoting it to a larger title.